terraform/azure/terraform.tfstate

```diff
@@ -0,0 +1,671 @@
+{
+  "version": 4,
+  "terraform_version": "1.2.7",
```

Comment:
remove the state file

Reply:
either keep the state file in local or push the state file to the azure blob store.